src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
Hmm, yes. But have we announced publicly that netcore will work on arm64? If we do that, then we have to change the api surface area anyways, hence there is no need to do this in parts. Moreover, on windows apis, we don't yet have an arm64 support. 

If this work needs to happen right now, then I guess we can throw a PNSE for Arm64, as if someone is going to hack to get this managed lib working on arm64, and use the property specifically to make a runtime decision to see if its arm64, then it's no useful to lie saying its arm 32 bit.